config/nrfconnect/chip-module/Kconfig

```diff
@@ -70,3 +70,145 @@ config CHIP_DEBUG_SYMBOLS
 	default y
 	help
 	  Build the application with debug symbols.
+
+# Factory data
+config CHIP_FACTORY_DATA_ENABLE
+	bool "Enables Factory Data support"
+	default n
+	help
+		Enables generating a factory data after build.
```

Comment:
I think that here and in all other places there is a problem with help description indentation.